packages/gatsby-cli/src/create-cli.ts

```diff
@@ -534,6 +540,38 @@ Creating a plugin:
         report.log(`Telemetry collection ${enabled ? `enabled` : `disabled`}`)
       }),
     })
+    .command({
+      command: `config`,
```

Comment:
Should we do something like instead?
```suggestion
      command: `config set <option> <value>`,
```

Reply:
So I was debating this. The only other command with that structure is `gatsby new`, everything else is flag based. But I'm not strongly opinionated either way.

Reply:
I'm not opinionated on this. `gatsby config` doesn't indicate that I want to set something, that's all